src/App.js

```diff
@@ -1,16 +1,23 @@
+import React from 'react';
+import {
+  BrowserRouter, Routes, Route,
+} from 'react-router-dom';
 import CalcDisplay from './Components/calculator';
 import ShowQuotes from './Components/quote';
+import Navbar from './Components/Navbar';
+import Home from './Components/HomePage';
 
 function App() {
   return (
-    <div className="App">
-      <div className="main">
-        <CalcDisplay />
-      </div>
-      <div className="main">
-        <ShowQuotes />
-      </div>
-    </div>
+    <BrowserRouter>
+      <Navbar />
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<CalcDisplay />} />
+        <Route path="/quote" element={<ShowQuotes />} />
+      </Routes>
+    </BrowserRouter>
```

Comment:
- [ ] As required for the project, kindly ensure that general layout of your app matchs the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). Right now it is impossible to see `Navbar` on each page unless user scrols horizonataly.
### Your Design
![Screenshot from 2023-08-16 03-27-07](https://github.com/Wils-FOURR-speed/math-magician/assets/83042742/79e03aaf-7f8f-4cb3-8966-67c79936494e)

### Expected wireframes
![Screenshot from 2023-08-16 03-29-42](https://github.com/Wils-FOURR-speed/math-magician/assets/83042742/81a59370-8ef4-4cce-bdfd-52f1f4fe2091)

Reply:
@tresorsawasawa Hi, please we were asked to personalize the website's styling.

Reply:
This change has been done well